modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -309,52 +309,49 @@ function update( $new_instance, $old_instance ) {
 		$instance = array();
 
 		$instance['title'] = sanitize_text_field( $new_instance['title'] );
-		$instance['use_filters'] = empty( $new_instance['use_filters'] ) ? '0' : '1';
 		$instance['search_box_enabled'] = empty( $new_instance['search_box_enabled'] ) ? '0' : '1';
 		$instance['user_sort_enabled'] = empty( $new_instance['user_sort_enabled'] ) ? '0' : '1';
 		$instance['sort'] = $new_instance['sort'];
 		$instance['post_types'] = empty( $new_instance['post_types'] ) || empty( $instance['search_box_enabled'] )
 			? array()
 			: array_map( 'sanitize_key', $new_instance['post_types'] );
 
-		if ( $instance['use_filters'] ) {
-			$filters = array();
-			foreach ( (array) $new_instance['filter_type'] as $index => $type ) {
-				$count = intval( $new_instance['num_filters'][ $index ] );
-				$count = min( 50, $count ); // Set max boundary at 20
-				$count = max( 1, $count );  // Set min boundary at 1
-
-				switch ( $type ) {
-					case 'taxonomy':
-						$filters[] = array(
-							'name' => sanitize_text_field( $new_instance['filter_name'][ $index ] ),
-							'type' => 'taxonomy',
-							'taxonomy' => sanitize_key( $new_instance['taxonomy_type'][ $index ] ),
-							'count' => $count,
-						);
-						break;
-					case 'post_type':
-						$filters[] = array(
-							'name' => sanitize_text_field( $new_instance['filter_name'][ $index ] ),
-							'type' => 'post_type',
-							'count' => $count,
-						);
-						break;
-					case 'date_histogram':
-						$filters[] = array(
-							'name' => sanitize_text_field( $new_instance['filter_name'][ $index ] ),
-							'type' => 'date_histogram',
-							'count' => $count,
-							'field' => sanitize_key( $new_instance['date_histogram_field'][ $index ] ),
-							'interval' => sanitize_key( $new_instance['date_histogram_interval'][ $index ] ),
-						);
-						break;
-				}
+		$filters = array();
+		foreach ( (array) $new_instance['filter_type'] as $index => $type ) {
```

Comment:
I'm getting an undefined index notice here: 

```
[29-Jan-2018 23:11:06 UTC] PHP Notice:  Undefined index: filter_type in wp-content/plugins/jetpack/modules/search/class.jetpack-search-widget-filters.php on line 320
```

Reply:
I think I fixed this